App.js

```diff
@@ -1,14 +1,17 @@
 const express = require('express'); //require express module
 const linebot = require('linebot');
+const app = express(); //creates an Express application
+const http = require('http');
+// for test
+global.request = require('supertest');
+request = request(app); 
```

Comment:
The above two lines should be moved to the test file (test/App.js)

Reply:
will fix it.